index.html

```diff
@@ -1 +1,944 @@
+                          </article>
+
+                          <article class="col-md-4">
+                              <div class="wrap">
+                                  <a href="#"><img src="https://d.newsweek.com/en/full/1513417/liverpool.jpg?w=397&h=265&f=f9441080ace170974365ba1168c6376d" class="img-fluid" alt=""></a>
+                              </div>
+                              <div class="article-info">
+                                  <h3>Liverpool vs. Borussia Dortmund: Where to Watch, Live Stream, Team News</h3>
+                              </div>
+                          </article>
+
+                          <div class="col-md-4">
+                              <article class="feature2 top-column">
+                                  <div class="row">
+                                      <div class=" col-5"><img src="https://d.newsweek.com/en/full/1513162/seattle-sounders.jpg?w=135&h=90&f=da24606b3a84bebcae77c85c4cd1b699" alt=""></div>
+                                      <div class="col-7">
+                                          <p class="info">Video: Heartwarming Moment 8-year-old Cancer Patient Starts for Sounders</p>
+                                      </div>
+                                  </div>
+                              </article>
+
+                              <article class="feature2">
+                                  <div class="row">
+                                      <div class=" col-5"><img src="https://d.newsweek.com/en/full/1513397/tiger-woods-open.jpg?w=135&h=90&l=0&t=0&f=1706d1c2af3b5c666dcd6d60bc97126f" alt=""></div>
+                                      <div class="col-7">
+                                          <p class="info">When Does Tiger Tee Off on Friday? The Open Second Round TV, Live Stream</p>
+                                      </div>
+                                  </div>
+                              </article>
+
+                              <article class="feature2">
+                                  <div class="row">
+                                      <div class=" col-5"><img src="https://d.newsweek.com/en/full/1513281/wwe-miz-interview-extreme-rules.jpg?w=135&h=90&f=b0f04c921424d19af395c71ed982fce7" alt=""></div>
+                                      <div class="col-7">
+                                          <p class="info">Miz Doesn't Want to be on PPV Without a Story</p>
+
+                                      </div>
+                                  </div>
+                              </article>
+                          </div>
+                      </div>
+                  </div>
+                </div>
+
+              </section>
+
+           </main>
+
+              <footer>
+                  <div class="footer-top">
+                      <div class="row">
+                          <div class="col-md-9 logo-footer">
+                              <img src="Images/newsweek-logo-2.png" alt="Newsweek Logo">
```

Comment:
`src=images/newsweek-logo-2.png` hope this helps 😄

Reply:
![Screen Shot 2019-07-22 at 5 31 57 PM](https://user-images.githubusercontent.com/31193823/61667131-2f06c700-aca7-11e9-86f9-79ac4a28e9f3.png)